tests/test_addon_detail.py

```diff
@@ -425,6 +426,29 @@ def test_compare_more_info_latest_version(selenium, base_url, variables):
     assert more_info_version == latest_version
 
 
+@pytest.mark.nondestructive
+def test_more_info_addon_tags(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # get the name of one of the tags related to this addon
+    tag_name = addon.more_info.addon_tags[0].text
+    addon.more_info.addon_tags[0].click()
+    # clicking on the tag opens a search results page with addons that share the same tag
+    same_tag_results = Search(selenium, base_url).wait_for_page_to_load()
+    # verifies that the results URL mention the tag name (encoded)
+    assert f'/tag/{urllib.parse.quote(tag_name)}/' in selenium.current_url
+    count = 0
+    # checking that search results do include the tag of the initial addon
+    while count <= 4:
```

Comment:
This will always be 4?

Reply:
@jrbenny35 no, we don't know what the exact number (`count`) of results might be , but I'm cross checking only with the first items in the list. I'm assuming there will always be more than 5 results for the same tag, so I haven't taken into account the possibility of running into an `list index out of range` exception. I could try to catch that exception though, just to be sure.